### ISSUE TITLE
# on valid_dataloaders for train valid_gt_encoder 

### ISSUE THREAD

**Issue by SolicTous** [xuebinqin/DIS#140]
1) Why you commented `valid_gt_encoder `validation from `valid_dataloaders`, and made it on `train_dataloaders`?
2) And here is commented` net.eval()`, so what is the reason?

**Comment by Maozhentan2024** [comment]
2.Not calling net.eval(): The model remains in training mode during validation, which may lead to unstable model outputs because Dropout and Batch Normalization layers operate according to the training mode.
